SD-K9/src/app/pages/app-settings/app-settings.page.ts

```diff
@@ -9,11 +10,27 @@ import { NavParams } from '@ionic/angular';
 })
 export class AppSettings implements OnInit {
 
-  constructor(private modalController: ModalController) { }
+  constructor(private modalController: ModalController, public themeSwitcher: ThemeSwitcherService, public alertController: AlertController) { }
 
   ngOnInit() {
   }
   async closeModal() {
     await this.modalController.dismiss();
   }
+
+  async changeTheme(){
+    this.themeSwitcher.cycleTheme();
+  }
+
+  async presentAlert() {
+    const alert = await this.alertController.create({
+      header: 'Alert',
+      subHeader: 'Subtitle',
+      message: 'It is NavConU\'s policy to respect your privacy regarding any information we may collect while operating our website. This Privacy Policy applies to www.NavConU.com (hereinafter, "us", "we", or "www.NavConU.com"). We respect your privacy and are committed to protecting personally identifiable information you may provide us through the Website. We have adopted this privacy policy ("Privacy Policy") to explain what information may be collected on our Website, how we use this information, and under what circumstances we may disclose the information to third parties. This Privacy Policy applies only to information we collect through the Website and does not apply to our collection of information from other sources.',
```

Comment:
Can you put the message in a config file?

Reply:
Yes that is a good idea, I'll look into it